cli/integration/tests/waiter/test_cli.py

```diff
@@ -2393,23 +2392,27 @@ def __test_signal(self, get_possible_instances_fn, signal_type=None, signal_flag
             stdout = cli.stdout(cp)
             self.assertEqual(0, cp.returncode, cp.stderr)
             self.assertIn(f'Successfully sent {signal_type_output} to', stdout)
+            cli_output = stdout.split()
+            for i,w in enumerate(cli_output):
+                if w == "instance":
+                    killed_instance_id = cli_output[i+1]
```

Comment:
You need an assertion outside the loop for `killed_instance_id` not being `None`.